Resources/Prototypes/_White/Catalog/uplink_catalog.yml

```
@@ -52,6 +52,23 @@
   saleLimit: 1
 
+- type: listing
+  id: UplinkHardlightSpearImplanter
+  name: uplink-hardlight-spear-implant-name
+  description: uplink-hardlight-spear-implant-desc
+  icon: { sprite: /Textures/_White/Objects/Weapons/Melee/Spear/hardlight_spear.rsi, state: spear }
+  productEntity: HardlightSpearImplanter
+  cost:
+    Telecrystal: 12
+  categories:
+  - UplinkImplants
```

yamllint

[warning] wrong indentation: expected 4 but found 2

(indentation)